src/Robo/Commands/Composer/ComposerCommand.php

```diff
@@ -61,4 +62,37 @@ public function requirePackage($package_name, $package_version, $options = ['dev
     return $result;
   }
 
+  /**
+   * Performs a composer install.
+   *
+   * @param bool $dev
+   *   Whether or no dev packages should be installed.
+   *
+   * @return bool
+   *   If the command was successful.
+   *
+   * @command composer:install
+   */
+  public function install($dev = TRUE) {
```

Comment:
I think we want to use dev as an option rather than an argument. 

```
$options = ['dev' => TRUE]
```

See https://github.com/consolidation/annotated-command#option-default-values